app/controllers/course/leaderboards_controller.rb

```diff
@@ -3,6 +3,7 @@ class Course::LeaderboardsController < Course::ComponentController
   before_action :check_component
   before_action :load_settings
   before_action :add_leaderboard_breadcrumb
+  helper Course::Achievement::ControllerHelper.name.sub(/Helper$/, '')
```

Comment:
I'm starting to think that each of these helpers should include a common module instead. thoughts?